tests/layer_tests/tensorflow_tests/test_tf_ReduceArithmeticOps.py

```diff
@@ -50,3 +50,51 @@ def test_reduce(self, params, operation, keep_dims, ie_device, precision, ir_ver
                                            use_legacy_frontend=use_legacy_frontend),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_legacy_frontend=use_legacy_frontend)
+
+
+class TestComplexProd(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'param_real:0' in inputs_info, "Test error: inputs_info must contain `param_real`"
+        assert 'param_imag:0' in inputs_info, "Test error: inputs_info must contain `param_imag`"
+        x_shape = inputs_info['param_real:0']
+        inputs_data = {}
+        inputs_data['param_real:0'] = np.random.randint(-10, 10, x_shape).astype(np.float32)
```

Comment:
please use `rng` with static seed to avoid sporadic issue. Check other tests with it.